packages/framework-core/src/sensor/health/health-indicators/booster-health-indicator.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  BoosterConfig,
+  HealthIndicatorResult,
+  HealthIndicatorMetadata,
+  HealthStatus,
+} from '@boostercloud/framework-types'
+import { boosterVersion } from './booster-version'
+
+export class BoosterHealthIndicator {
+  public async health(
+    config: BoosterConfig,
+    healthIndicatorMetadata: HealthIndicatorMetadata
+  ): Promise<HealthIndicatorResult> {
+    try {
+      const result: HealthIndicatorResult = {
+        status: await this.isUp(config),
+      }
+      if (healthIndicatorMetadata.healthIndicatorConfiguration.details) {
+        result.details = {
+          boosterVersion: boosterVersion(config),
+        }
+      }
+      return result
+    } catch (e) {
+      return { status: HealthStatus.DOWN, details: e }
+    }
+  }
+
+  private async isUp(config: BoosterConfig): Promise<HealthStatus> {
+    const graphqlUp = await config.provider.sensor.isGraphQLFunctionUp(config)
+    const eventsCatalog = await config.provider.sensor.isEventCatalogUp(config)
+    return graphqlUp && eventsCatalog ? HealthStatus.UP : HealthStatus.DOWN
```

Comment:
Shouldn't we also include here the read models health status? I understood this health indicator as a general one that confirms everything is OK